website/docs/aggregation-operations/aggregation-pipeline-and-commands.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 1
+---
+
+# Aggregation pipeline and commands
+
+Aggregation operations involve performing various operations on a large number of data records, such as data grouping, sorting, restructuring, or modifying.
+These operations pass through one or more stages, which make up a pipeline.
+
+![aggregation stages](/img/docs/aggregation-stages.jpg)
+
+Each stage acts upon the returned documents of the previous stage, starting with the input documents.
+As shown above, the documents pass through the pipeline with the result of the previous stage acting as input for the next stage, going from `$match` => `$group` => `$sort` stage.
+
+For example, say you have the following documents in a `sales` collection:
+
+```js
+;[
```

Comment:
Should this one be `json5`, not `js`?

Reply:
It should still be `js` but turned it into an `insert` statement now.